Cargo.toml

```diff
@@ -24,7 +24,7 @@ chrono = "0.4.38"
 duckdb = { git = "https://github.com/paradedb/duckdb-rs.git", features = [
   "bundled",
   "extensions-full",
-], rev = "e532dd6" }
+], rev = "c2f9e2010e326de21126e90dc24da47e0a962cb0" }
```

Comment:
Where is this commit from?

Reply:
Note that we use our own fork of duckdb-rs. It is up-to-date, though.

Reply:
yes, it's the latest commit from paradedb/duckdb-rs